From 9391d44430884cd28863c63206360258b7d6169b Mon Sep 17 00:00:00 2001
From: Vishal Sagar <vishal.sagar@xilinx.com>
Date: Thu, 1 Oct 2020 05:25:18 -0700
Subject: [PATCH 1687/1851] v4l: xilinx: sdirxss: Update HDR control type

commit ad94d03f1d6b2b8014f3e45317ba5a84508fc9d8 from
https://github.com/Xilinx/linux-xlnx.git

Update the HDR control type as defined in videodev2.h which is updated
to remove coverity warning.

Signed-off-by: Vishal Sagar <vishal.sagar@xilinx.com>
Reviewed-by: Hyun Kwon <hyun.kwon@xilinx.com>
State: pending
Signed-off-by: Yaliang Wang <Yaliang.Wang@windriver.com>
---
 drivers/media/platform/xilinx/xilinx-sdirxss.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/media/platform/xilinx/xilinx-sdirxss.c b/drivers/media/platform/xilinx/xilinx-sdirxss.c
index fd571e8c77da..8b7c335bea13 100644
--- a/drivers/media/platform/xilinx/xilinx-sdirxss.c
+++ b/drivers/media/platform/xilinx/xilinx-sdirxss.c
@@ -2215,7 +2215,7 @@ static const struct v4l2_ctrl_config xsdirxss_ctrls[] = {
 		.ops	= &xsdirxss_ctrl_ops,
 		.id	= V4L2_CID_METADATA_HDR,
 		.name	= "HDR Controls",
-		.type	= 0x0106,
+		.type	= V4L2_CTRL_TYPE_HDR,
 		.min	= 0x8000000000000000,
 		.max	= 0x7FFFFFFFFFFFFFFF,
 		.step	= 1,
-- 
2.31.1

